setup_script.sh

```diff
@@ -34,6 +34,78 @@ if ! command -v "node" &>/dev/null; then
     exit
 fi
 
+###
+# check for the existence of required files in the Distro and camicroscope repository
+###
+echo "Checking for required files"
+
+if [[ ! -d ../Distro/db ]]
+then echo "../Distro/db does not exist"
+     exit
+fi
+
+if [[ ! -d ../Distro/config ]]
+then echo "../Distro/config does not exist"
+     exit
+fi
+
+if [[ ! -d ../Distro/jwt_keys ]]
+then echo "../Distro/jwt_keys does not exist"
+     exit
+fi
+
+if [[ ! -d ../caMicroscope ]]
+then echo "../caMicroscope does not exist"
+     exit
+fi
```

Comment:
If you're planning on requiring this to be in the Distro context, I'd change the readme info for the setup script.
If not, then maybe you could simply check if camicroscope's static files aready exist here first, then clone a fresh copy if they dont?

Reply:
I feel the entire caMicroscope repo would be required. I have added the copy lines in the setup_script based on develop.yml file of the Distro repository

Reply:
The setup script section of the readme should describe the desired directory structure, at the least.